enterprise/web.jsf.editor/src/org/netbeans/modules/web/jsf/editor/facelets/CompositeComponentLibrary.java

```diff
@@ -89,7 +90,7 @@ public LibraryType getType() {
 
     @Override
     public String getDefaultNamespace() {
-        return LibraryUtils.getCompositeLibraryURL(getLibraryName(), support.getJsfSupport().isJsf22Plus());
+        return LibraryUtils.getCompositeLibraryURL(getLibraryName(), support.getJsfSupport().getJsfVersion().isAtLeast(JsfVersion.JSF_2_2));
```

Comment:
I'm surprised. Did they really go to URNs and did not change the composite component namespace?

Reply:
Actually I did not checked the context of that change tbh. I just wanted to get rid of the `isJsf**Plus` methods and use something more generic.
This was one call where such a method was used so I replaced it with the corresponding call.